external/model-preparation-algorithm/configs/detection/resnet50_vfnet_cls_incr/tile_pipeline.py

```diff
@@ -0,0 +1,85 @@
+dataset_type = "CocoDataset"
+
+img_size = (1024, 1024)
+
+tile_cfg = dict(
+    tile_size=400, min_area_ratio=0.9, overlap_ratio=0.2, iou_threshold=0.45, max_per_img=1500, filter_empty_gt=True
+)
+
+img_norm_cfg = dict(mean=(103.53, 116.28, 123.675), std=(1.0, 1.0, 1.0), to_rgb=False)
```

Comment:
Could you align normalize vactor and rgb/bgr?

Reply:
Done